server.js

```diff
@@ -1,25 +1,19 @@
-app.put('/slambook/:id', async (req, res) => {
+app.delete('/slambook/:id', async (req, res) => {
   const entryId = req.params.id;

```

Comment:
The entryId parameter is missing in the findByIdAndDelete() function. It should be req.params.id